installer/manifests/keptn/charts/continuous-delivery/templates/continuous-deployment.yaml

```diff
@@ -278,7 +278,14 @@ spec:
           {{- include "continuous-delivery.livenessProbe" . | nindent 10 }}
           imagePullPolicy: IfNotPresent
           ports:
-          - containerPort: 8080
+            - containerPort: 8080
+          resources:
+            requests:
+              memory: "64Mi"
+              cpu: "50m"
+            limits:
+              memory: "128Mi"
```

Comment:
In my monitoring jmeter-service shows up that it needs up to 1.5 GB memory depending on the use-case
![image](https://user-images.githubusercontent.com/56065213/98930701-46da0e00-24dd-11eb-90af-b1768953b515.png)

Reply:
FYI, a simple "new-artifact" on 0.7.2 leads to roughly 500 MB memory usage

Reply:
Thanks for the insights. Well, then my resource limits are too (too) low. :) 

What is your suggestion on that?

Reply:
As discussed, I will define the `requests` but not the `limits`.